docs/bundles/SyliusResourceBundle/routing.rst

```diff
@@ -232,3 +232,63 @@ If you want to change that behavior, use the following configuration:
             alias: app.book
             redirect: update
         type: sylius.resource
+
+API Versioning
+--------------
+
+In order to versionate your API responses, you can configure a version of returned responses.
+If you would like to return only second version of your responses, use the following snippet:
+
+.. code-block:: yaml
+
+    app_book:
+        resource: |
+            alias: app.book
+            serialization_version: 2
+        type: sylius.resource_api
+
+Whats more, you can use a path variable to dynamically change your request. You can achieve this by setting a path prefix when importing file or specify it in a path option.
+
+.. code-block:: yaml
+
+    app_book:
+        resource: |
+            alias: app.book
+            serialization_version: $version
+        type: sylius.resource_api
+
+.. note::
+
+    Remember, that a dynamically resolved `books` prefix is not longer available if you will specify ``path``, and it has to be defined manually.
+
+Using a Custom Identifier
+-------------------------
+
+As you could notice the generated routing resolve resources by ``id`` field. But sometimes it is more convenient to use a custom identifier field instead, lets say ``code`` (or any other field which can uniquely identify your resource).
```

Comment:
`resolve` => `resolves`, ```the ``id`` ``` 
how 'bout `(or any other field of your choice which can ...)`